lma-integration/run-tests

```diff
@@ -1,5 +1,14 @@
 #!/bin/bash
 
+set -e
+
+progdir=$(dirname "$(readlink -f "${0}")")
+
+# We have to build the image of the test container first.
+cd "${progdir}/tests/"
+docker build . -t localtest/lma-integration-tests
+cd -
```

Comment:
Could we just change the build context instead to read `docker build ${progdir}/tests/ -t localtest/lma-integration-tests`?

Reply:
I wasn't sure whether I could pass the build directory directly to `docker build` or not (because we use a `COPY . .` statement inside the Dockerfile, so I was afraid that it was going to affected by not chdir'ing).  Thanks for the tip; I've tested and verified that it indeed works :-).